release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -165,6 +165,20 @@ Netlink messages dropped by OVS kernel module due to netlink socket buffer overf
 [id="ocp-4-13-scalability-and-performance"]
 === Scalability and performance
 
+[id="ocp-4-13-NUMA-scheduler-ga"]
+==== NUMA-aware scheduling with the NUMA Resources Operator is generally available
+
+With {product-title} {product-version}, NUMA-aware scheduling with the NUMA Resources Operator is generally available. This was a Technology Preview feature in {product-title} 4.10, 4.11, and 4.12. 
+
+The NUMA Resources Operator deploys a NUMA-aware secondary scheduler that makes scheduling decisions for workloads based on a complete picture of available NUMA zones in clusters. This ensures that latency-sensitive workloads are processed in a single NUMA zone for maximum efficiency and performance. 
+
+This update also adds the following features:
```

Comment:
What about 
```suggestion
With this update you can:
```
or
```suggestion
This update adds the following features:
```
I would remove the "also" here because it is not needed.